apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.IntegrationTests/ReadTests.cs

```diff
@@ -372,10 +372,35 @@ private class CredentialWrapper : ITokenAccess
         }
 
         [Fact]
+        public async Task CommandTimeout()
+        {
+            long result = 0;
+            var e = await Assert.ThrowsAsync<SpannerException>(
+                async () =>
+                {
+                    using (await _testFixture.GetTestDatabaseConnectionAsync())
+                    {
+                        using (var connection =
+                            new SpannerConnection(_testFixture.ConnectionString))
+                        {
+                            var cmd =
+                                connection.CreateSelectCommand("SELECT 1");
+                            cmd.CommandTimeout = 0;
```

Comment:
it definitely suggests that in the msdn docs doesnt it.
also, it would make sense to change spanneroptions to also be seconds instead of timespan because you'd otherwise get strange inconsistencies when the value was carried over (either CommandTimeout would lie when it returns its value due to loss of precision or the actual timeout used would lose precision, making TimeSpan uselss).  So I think I'll have to return to my original instinct in SpannerOptions (I think it was this way at some point before).
It will surely make the timeout tests more challenging as 1 second isnt enough time to reliably trigger a timeout.  thinking...